onnxruntime/test/providers/qnn/gather_op_htp_test.cc

```
@@ -131,11 +131,15 @@ TEST_F(QnnHTPBackendTests, GatherOp_IndicesDynamicInt32_Axis0) {
                                        ExpectedEPNodeAssignment::All);
 }
 
+// disabled for QNN 2.28.0.241029 failed for accuracy validation
```

we should report these new failures

---

yes